### PR TITLE
[AD-145] Adding topsite dismissals to newtab_* jobs.

### PR DESCRIPTION
**Pull request by sergiosonline**
Checklist for reviewer:

- [ ] Commits should reference a bug or github issue, if relevant (if a bug is referenced, the pull request should include the bug number in the title).
- [ ] If the PR comes from a fork, trigger integration CI tests by running the [Push to upstream workflow](https://github.com/mozilla/bigquery-etl/actions/workflows/push-to-upstream.yml) and provide the `<username>:<branch>` of the fork as parameter. The parameter will also show up
in the logs of the `manual-trigger-required-for-fork` CI task together with more detailed instructions.
- [ ] If adding a new field to a query, ensure that the schema and dependent downstream schemas have been updated.
- [ ] When adding a new derived dataset, ensure that data is not available already (fully or partially) and recommend extending an existing dataset in favor of creating new ones. Data can be available in the [bigquery-etl repository](https://github.com/mozilla/bigquery-etl), [looker-hub](https://github.com/mozilla/looker-hub) or in [looker-spoke-default](https://github.com/mozilla/looker-spoke-default/tree/e1315853507fc1ac6e78d252d53dc8df5f5f322b).

For modifications to schemas in restricted namespaces (see [`CODEOWNERS`](https://github.com/mozilla/bigquery-etl/blob/main/CODEOWNERS)):
- [ ] Follow the [change control procedure](https://docs.google.com/document/d/1TTJi4ht7NuzX6BPG_KTr6omaZg70cEpxe9xlpfnHj9k/edit#heading=h.ttegrcfy18ck)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/DENG-2297)

### CODE REVIEW

**Review by curtismorales — COMMENTED** [review]
This looks correct and straighforward to me. Out of an abundance of caution, I'd also tag @ANich to take a look and make sure there aren't any gotchas here that we're not aware of

**Comment by sergiosonline** [comment]
@curtismorales : Do you know if there are any issues with circleci since yesterday? All the checks seemed to pass except for `test-sql`, but when I click for details it just stays loading.

File: sql/moz-fx-data-shared-prod/telemetry_derived/newtab_interactions_v1/query.sql
Comment (ANich) [inline sql/moz-fx-data-shared-prod/telemetry_derived/newtab_interactions_v1/query.sql:1]: This table was meant to be deprecated `newtab_visits_v1` has all the same information

**Review by ANich — COMMENTED** [review]
LGTM overall, left one comment. Tests need to be updated to include the new columns

File: sql/moz-fx-data-shared-prod/telemetry_derived/newtab_interactions_v1/query.sql
Comment (sergiosonline) [reply sql/moz-fx-data-shared-prod/telemetry_derived/newtab_interactions_v1/query.sql:1]: @ANich: I do agree that they share most of the info, but I hadn't heard of plans to deprecate `interactions`. Is there a doc or convo where this is formally contemplated?

If it's deprecated, then we'll need to update a number of Looker dashboards and most of the interaction metrics in `metric_hub`.

Comment (ANich) [reply sql/moz-fx-data-shared-prod/telemetry_derived/newtab_interactions_v1/query.sql:1]: Here's the original ticket that eventually led to `newtab_visits_v1` - https://mozilla-hub.atlassian.net/browse/DENG-796 Deprecating `newtab_interactions_v1` did get de-prioritized (at least for me), then I imagine dropped after the team working on this changed

Comment (sergiosonline) [reply sql/moz-fx-data-shared-prod/telemetry_derived/newtab_interactions_v1/query.sql:1]: @ANich / @curtismorales : Then I suggest we just go ahead with this change and assess deprecation later on when it becomes more pressing. @m-d-bowerman and I can start looking at some mitigation steps post deprecation, given how there are a lot of metrics in metric_hub that depend on interactions. There may also be other Looker dependencies.

**Review by curtismorales — APPROVED** [review]

**Pull request merged**